stacks_queues/stack.py

```diff
@@ -21,18 +22,25 @@ def pop(self):
             The Stack is empty.
             returns None
         """
-        pass
+        if self.store == None:
+            raise StackEmptyException("The stack is already empty!")
+        temp = self.store.get_first()
+        self.store.remove_first()
+        return temp
 
     def empty(self):
```

Comment:
👀 This works, but the Linked List class has an `empty` method you could use as well